.codeclimate.yml

```diff
@@ -5,6 +5,6 @@ checks:
       threshold: 60
   method-complexity:
     config:
-      threshold: 8
+      threshold: 9
```

Comment:
I don't think this is needed any more, since you broke out some logic from `selectCell`

Reply:
Okay. Gonna try.